bamoe-ddl/postgresql/jobs-service.sql

```diff
@@ -0,0 +1,54 @@
+--
+-- Jobs service uses below tables
+--
+-- job_details              details of job instance being created
+-- job_service_management   used for clustering and to check lead instance
+--
+
+CREATE TYPE job_status AS ENUM (
+    'ERROR',
+    'EXECUTED',
+    'SCHEDULED',
+    'RETRY',
+    'CANCELED'
+);
+
+CREATE TYPE job_type AS ENUM (

```

Comment:
@josedee This can be removed, it was used in the v1 table version you removed.

Reply:
<img width="694" alt="image" src="https://github.com/user-attachments/assets/fb5e9cc9-e1e8-4941-8e91-c1aafe8e7b36">
Nice catch, I just verified with job_details table